docs/contributing-to-airbyte/writing-docs.md

```diff
@@ -64,7 +64,7 @@ To make complex changes or edit multiple files, edit the files on your local mac
    yarn start
    ```
 
-   Then navigate to [http://localhost:3000/](http://localhost:3000/). Whenever you make and save changes, you will see them reflected in the server. You can stop the running server in OSX/Linux by pressing `Ctrl-C` in the terminal.  
+   Then navigate to [http://localhost:3005/](http://localhost:3005/). Whenever you make and save changes, you will see them reflected in the server. You can stop the running server in OSX/Linux by pressing `Ctrl-C` in the terminal.  
```

Comment:
Should the `localhost:3000` below on line 76 also be changed to `localhost:3005`?

Reply:
Nope! I had the same thought, and actually had to go back and fix my initial rewrite after double-checking the docs directory's `package.json`: we pass a `--port 3005` flag to the dev server, but the prod-style one uses docusaurus's default of `3000`.